jep/200/README.adoc

```diff
@@ -127,12 +127,17 @@ Typically the rejection will also trigger an exception which gets displayed in s
 Plugin developers will be able to whitelist the classes in their libraries/plugins using
 the following approaches:
 
-0. `Jenkins-ClassFilter-Whitelisted` manifest entry (link:https://github.com/jenkinsci/lib-jenkins-maven-embedder/pull/15[example for Maven])
-0. By providing a custom `META-INF/hudson.remoting.ClassFilter` resource file in the plugin
-0. By implementing the `CustomClassFilter` extension point and implementing `permits()` methods
-
-The first two approaches do not require Jenkins core requirement update in plugins.
-Old Hudson plugins may also need additional manifest properties to be processed as Jenkins plugins by the engine.
+0. By providing a custom `META-INF/hudson.remoting.ClassFilter` resource file in their plugin
+0. By adding the `Jenkins-ClassFilter-Whitelisted=true` manifest entry (link:https://github.com/jenkinsci/lib-jenkins-maven-embedder/pull/15[example for Maven])
+** Old Hudson plugins **may** need an update to be processed as Jenkins plugins by the extensions mentioned above.
+** The manifest should have `Short-Name`, and any of `Plugin-Version`/`Jenkins-Version` entries
+** According to the analysis, Jenkins/Hudson plugin POMs offer these entries by default for versions newer than 1.320
```

Comment:
Sorry, where is this analysis? Is there an example of a plugin built against 1.320- that would not pass?

Reply:
I can confirm it for 1.368 and the Perforce Plugin. It bundles the required entries. IDGAF about plugins requiring something older

Reply:
So the comment is misleading. What you meant to say is that version 1.320 is known to be fine…and probably much older versions are fine, too. But the wording implies that it is known to be broken for certain older versions.